src/libs/model/index.ts

```diff
@@ -0,0 +1,213 @@
+          relation.properties.forEach(
+            property => (property.container = relation),
+          );
+        }
+        contents = [
+          ...contents,
+          ...(relation.properties ? relation.properties : []),
+        ];
+      } else if (element.type === OntoUMLType.CLASS_TYPE) {
+        const _class = element as IClass;
+
+        if (_class.properties) {
+          _class.properties.forEach(property => (property.container = _class));
+        }
+        if (_class.literals) {
+          _class.literals.forEach(literal => (literal.container = _class));
+        }
+        contents = [
+          ...contents,
+          ...(_class.properties ? _class.properties : []),
+          ...(_class.literals ? _class.literals : []),
+        ];
+      }
+
+      element.container = { type: current.type, id: current.id };
+    });
+
+    return contents;
+  }
+
+  /**
+   * Returns an IElement according to its id.
+   *
+   * @param elementId - Desired element's id.
+   */
+  getElementById(elementId: string): any {
+    return this.getElements().find(element => element.id === elementId);
+  }
+
+  checkInstanceOfIContainer(element: IElement): boolean {
+    return [
+      OntoUMLType.PACKAGE_TYPE,
+      OntoUMLType.CLASS_TYPE,
+      OntoUMLType.RELATION_TYPE,
+    ].includes(element.type);
+  }
+
+  checkInstanceOfIDecoratable(element: IElement): boolean {
+    return [
+      OntoUMLType.PROPERTY_TYPE,
+      OntoUMLType.CLASS_TYPE,
+      OntoUMLType.RELATION_TYPE,
+    ].includes(element.type);
+  }
+
+  checkInstanceOfIClassifier(element: IElement): boolean {
+    return [OntoUMLType.CLASS_TYPE, OntoUMLType.RELATION_TYPE].includes(
+      element.type,
+    );
+  }
+
+  injectFunctions(element: IElement, enableMemoization = true): void {
+    Object.keys(functions.IElement_functions).forEach(
+      (functionName: string) => {
+        element[functionName] = enableMemoization
+          ? memoizee(functions.IElement_functions[functionName])
+          : functions.IElement_functions[functionName];
+      },
+    );
+
+    if (element.hasIContainerType()) {
+      Object.keys(functions.IContainer_functions).forEach(
+        (functionName: string) => {
+          element[functionName] = enableMemoization
+            ? memoizee(functions.IContainer_functions[functionName])
+            : functions.IContainer_functions[functionName];
+        },
+      );
+    }
+
+    if (element.hasIClassifierType()) {
+      Object.keys(functions.IClassifier_functions).forEach(
+        (functionName: string) => {
+          element[functionName] = enableMemoization
+            ? memoizee(functions.IClassifier_functions[functionName])
+            : functions.IClassifier_functions[functionName];
+        },
+      );
+    }
+  }
+
+  ejectFunctions(element: IElement): void {
+    Object.keys(element).forEach((elementKey: string) => {
+      if (element[elementKey] instanceof Function) {
+        element[elementKey] = undefined;
```

Comment:
Replace for `delete element[elementKey]` You can check the difference [here](https://stackoverflow.com/questions/14967535/delete-a-x-vs-a-x-undefined)